1 Standard/025_Pikachu.yaml

```diff
@@ -73,3 +73,22 @@ ball: [Poke]
 nature: Docile
 evs: {hp: 0, atk: 0, def: 168, spA: 0, spD: 168, spe: 172}
 ivs: 31
+---
+species: Pikachu
+setname: Smash
+tags: [standard]
+item: [null]
+ability: [Static]
+moves:
+    - [Volt Tackle]
+    - [Skull Bash]
+    - [Quick Attack]
+    - [Thunder]
```

Comment:
bad move order

Reply:
Just so I know for the future, is there a general order? Like, I’m assuming primary STAB attack at the front and status moves at the end?

Reply:
generally the STABs/ strongest attacks vs a neutral, same defences target go in the top slots, the coverage moves, then "good" status moves like setup or crippling status, then niche status moves like foresight.